### PR TITLE
(#3519) NCIDS with and without title style home landing page.

### PR DESCRIPTION
**Pull request by smitahantziosps**
https://github.com/NCIOCPL/cgov-digital-platform/issues/3519 - Home & Landing Pages w/o Title
https://github.com/NCIOCPL/cgov-digital-platform/issues/3551 - Home & Landing Pages w/ Title

### CODE REVIEW

**Comment by smitahantziosps** [comment]
ODE Ready for QA Testing - https://ncigovcdode525.prod.acquia-sites.com/

File: docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/home_landing/field--node--field-landing-contents.html.twig [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{% set page_style = getHomeLandingStyle() %}
+{% if page_style == "ncid_without_title" %}
```
Comment (bryanpizzillo) [inline docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/home_landing/field--node--field-landing-contents.html.twig:2]: This should be with or without a title. Not just ncids_without_title. Also NCIDS is spelled wrong here.

File: docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/home_landing/node--cgov-home-landing--ncid-without-title--full.html.twig [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<div class="grid-container">
```
Comment (bryanpizzillo) [inline docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/home_landing/node--cgov-home-landing--ncid-without-title--full.html.twig:1]: You should not have grid container.

File: docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/home_landing/node--cgov-home-landing--ncid-with-title--full.html.twig [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{# Title Row #}
+<div class="text-center padding-y-6">
+    {% set hr_attribs = create_attribute({}) %}
+    {% if is_front == TRUE %}
+      {% set hr_attribs = hr_attribs.addClass('show-for-sr') %}
+    {% endif %}
+    <h1 {{hr_attribs}}>{{ node.label }}</h1>
+</div>
+{# End Title Row#}
+<div class="grid-container">
```
Comment (bryanpizzillo) [inline docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/home_landing/node--cgov-home-landing--ncid-with-title--full.html.twig:10]: You should not have grid container

File: docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/home_landing/node--cgov-home-landing--ncid-with-title--full.html.twig [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{# Title Row #}
+<div class="text-center padding-y-6">
+    {% set hr_attribs = create_attribute({}) %}
+    {% if is_front == TRUE %}
+      {% set hr_attribs = hr_attribs.addClass('show-for-sr') %}
```
Comment (bryanpizzillo) [inline docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/home_landing/node--cgov-home-landing--ncid-with-title--full.html.twig:5]: This conditional should not be needed. The front page will just use the template without a title.

File: docroot/profiles/custom/cgov_site/modules/custom/cgov_home_landing/config/install/field.storage.node.field_page_style.yml [hunk 5]
```diff
@@ -19,6 +19,12 @@ settings:
     -
       value: dceg_connect
       label: 'DCEG Connect'
+    -
+      value: ncid_without_title
```
Comment (bryanpizzillo) [inline docroot/profiles/custom/cgov_site/modules/custom/cgov_home_landing/config/install/field.storage.node.field_page_style.yml:23]: This should be ncids in both values.

**Review by bryanpizzillo — CHANGES_REQUESTED** [review]

File: docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/home_landing/field--node--field-landing-contents.html.twig (hunk 1 above)
Comment (smitahantziosps) [reply docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/home_landing/field--node--field-landing-contents.html.twig]: Done

File: docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/home_landing/node--cgov-home-landing--ncid-without-title--full.html.twig (hunk 2 above)
Comment (smitahantziosps) [reply docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/home_landing/node--cgov-home-landing--ncid-without-title--full.html.twig]: Done

File: docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/home_landing/node--cgov-home-landing--ncid-with-title--full.html.twig (hunk 3 above)
Comment (smitahantziosps) [reply docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/home_landing/node--cgov-home-landing--ncid-with-title--full.html.twig]: Done

File: docroot/profiles/custom/cgov_site/modules/custom/cgov_home_landing/config/install/field.storage.node.field_page_style.yml (hunk 5 above)
Comment (smitahantziosps) [reply docroot/profiles/custom/cgov_site/modules/custom/cgov_home_landing/config/install/field.storage.node.field_page_style.yml]: Done. thanks

**Pull request closed without merging**